src/content/docs/setup/install.mdx

```diff
@@ -104,6 +104,24 @@ Running it via Docker can be nice for just trying out kubecolor:
 docker run --rm -it -v $HOME/.kube:/home/nonroot/.kube:ro ghcr.io/kubecolor/kubecolor get pods
 ```
 
+== asdf
+
+For users using the [asdf](https://github.com/asdf-vm/asdf) package manager.
+
+```sh
+asdf plugin-add kubecolor https://github.com/dex4er/asdf-kubecolor.git
+asdf install kubecolor latest
+asdf global kubecolor latest
+```
+
+== mise
```

Comment:
Please use the same header style as the rest of the file


```suggestion
## mise
```
same for the asdf header